tests/src/test/java/alluxio/client/fs/UfsSyncIntegrationTest.java

```diff
@@ -863,14 +863,20 @@ public void deleteUfsFileGetStatus() throws Exception {
 
     // delete the file and wait a bit
     new File(ufsPath("/delete/file")).delete();
-    CommonUtils.sleepMs(2000);
+    CommonUtils.sleepMs(3000);
 
     // getStatus (not listStatus) on the root, with a shorter interval than the sleep.
     // This will sync that directory. The sync interval has to be long enough for the internal
     // syncing process to finish within that time.
     mFileSystem.getStatus(new AlluxioURI(alluxioPath("/delete")), GetStatusPOptions.newBuilder()
         .setCommonOptions(
-            FileSystemMasterCommonPOptions.newBuilder().setSyncIntervalMs(1000).build()).build());
+            FileSystemMasterCommonPOptions.newBuilder().setSyncIntervalMs(2000).build()).build());
```

Comment:
This is just a note as breadcrumb to who traces back to this PR.

`GetStatus(dir)` used to trigger a full recursive sync on the dir. Now this PR changes it to NOT doing that, and instead only sync on the dir itself. This test is therefore updated to adapt to the new behavior.

This is theoretically a behavior change. But I don't think anyone is relying on the behavior where GetStatus(dir) triggers a full recursive sync. That would be someone GetStatus(dir) then directly accessing children files without a ListStatus(dir) to know what the children are first. That is a rare use and I'd recommend such users change that GetStatus(dir) call to ListStatus(dir) to sync the dir.